src/celeritas/ext/detail/GeantPhysicsList.cc

```diff
@@ -215,37 +215,38 @@ void GeantPhysicsList::add_gamma_processes()
 /*!
  * Add EM processes for electrons and positrons.
  *
- * | Processes                    | Model classes             |
- * | ---------------------------- | --------------------------|
- * | Pair annihilation            | G4eeToTwoGammaModel       |
- * | Ionization                   | G4MollerBhabhaModel       |
- * | Bremsstrahlung (low E)       | G4SeltzerBergerModel      |
- * | Bremsstrahlung (high E)      | G4eBremsstrahlungRelModel |
- * | Coulomb scattering           | G4eCoulombScatteringModel |
- * | Multiple scattering (low E)  | G4UrbanMscModel           |
- * | Multiple scattering (high E) | G4WentzelVIModel          |
+ * | Processes                    | Model classes                |
+ * | ---------------------------- | ---------------------------- |
+ * | Pair annihilation            | G4eeToTwoGammaModel          |
+ * | Ionization                   | G4MollerBhabhaModel          |
+ * | Bremsstrahlung (low E)       | G4SeltzerBergerModel         |
+ * | Bremsstrahlung (high E)      | G4eBremsstrahlungRelModel    |
+ * | Coulomb scattering           | G4eCoulombScatteringModel    |
+ * | Multiple scattering (low E)  | G4UrbanMscModel              |
+ * | Multiple scattering (low E)  | G4GoudsmitSaundersonMscModel |
+ * | Multiple scattering (high E) | G4WentzelVIModel             |
  *
  * \note
- * - Bremsstrahlung models are selected manually at compile time using
- *   \c GeantBremsstrahlungProcess::ModelSelection and need to be updated
- *   accordingly.
  * - Coulomb scattering and multiple scattering (high E) are currently
  *   disabled.
  */
 void GeantPhysicsList::add_e_processes(G4ParticleDefinition* p)
 {
     auto* physics_list = G4PhysicsListHelper::GetPhysicsListHelper();
 
-    if (p == G4Positron::Positron())
+    if (options_.annihilation)
     {
-        // e+e- annihilation: G4eeToTwoGammaModel
-        physics_list->RegisterProcess(new G4eplusAnnihilation(), p);
+        if (p == G4Positron::Positron())
+        {
+            // e+e- annihilation: G4eeToTwoGammaModel
+            physics_list->RegisterProcess(new G4eplusAnnihilation(), p);
 
-        CELER_LOG(debug) << "Loaded pair annihilation with "
-                            "G4eplusAnnihilation";
+            CELER_LOG(debug) << "Loaded pair annihilation with "
+                                "G4eplusAnnihilation";
+        }
     }
 
-    if (true)
+    if (options_.ionization)
```

Comment:
If ionization is off, do we still correctly get dE/dx from brems? Or is our along-step data zeroed out?

Reply:
Ping @whokion : can we correctly get radiative slowing-down data when disabling ionization?

Reply:
If I recall correctly, with ionization disabled we'll still get the (brems contribution to) dE/dx from the brems process: in that case `IsIonisationProcess()` will be true for brems.